USERGUIDE.md

```diff
@@ -189,9 +200,9 @@ Once the measurements are stored in the database, we will want to display the re
 
 [Grafana](http://grafana.com/) provides a dashboard and alerting interface.
 
-Head to the Grafana interface, if you are using **minikube**, you can use the following command :
+Head to the Grafana web interface by taping in your browser the `grafana.traefikIngress.host`. E.g. :
 ```
-minikube service -n fadi fadi-grafana
+http(s)://grafana.test.local
```

Comment:
should be a clickable url, not sure why the (s), I'd suggest to remove it if both work